NitroxPatcher/Patches/Dynamic/FootstepSounds_OnStep_Patch.cs

```diff
@@ -61,4 +80,28 @@ private static float CalculateVolume(float originalVolume, FootstepSounds instan
         Resolve<FMODWhitelist>().TryGetSoundData(EXO_STEP_SOUND_PATH, out SoundData soundData);
         return soundData.Radius;
     });
+
+    private static void SendFootstepPacket(FMODAsset asset)
+    {
+        if (Resolve<LocalPlayer>().PlayerId.HasValue)
+        {
+            FootstepPacket.StepSounds assetIndex;
+            switch (asset.path)
+            {
+                case PRECURSOR_STEP_SOUND_PATH:
+                    assetIndex = FootstepPacket.StepSounds.PRECURSOR;
+                    break;
+                case METAL_STEP_SOUND_PATH:
+                    assetIndex = FootstepPacket.StepSounds.METAL;
+                    break;
+                case LAND_STEP_SOUND_PATH:
+                    assetIndex = FootstepPacket.StepSounds.LAND;
+                    break;
+                default:
+                    return;
+            }
```

Comment:
Isn't this closing braked to early? If so, maybe revert the if to an early return

Reply:
you highlighted the closing bracket for the switch/case statement but I inverted the if anyways because I think I do like it better that way

Reply:
You are right, mb.